footer/dependent-css/footer-brand.css

```diff
@@ -16,6 +16,14 @@
 	vertical-align: top;
 }
 
+.esri-footer-brand-link img {
```

Comment:
@dlewis-esri this should probably be it's own class instead of a nested selector attached to a raw `img` element. Something like `.esri-footer-brand-image`.

Reply:
@paulcpederson  i tend to disagree - you don't need to put classes on everything it just makes your markup bigger, if your following a well defined html structure IMHO this is perfectly acceptable

Reply:
I just meant to follow the practices used in this repo.